### PR TITLE
Add "Contributing" and "Requirements" to README.md

### PR DESCRIPTION
**Pull request by ExclusiveKnight**
I have added "Contributing" to README.md because there will be forks of the main repository, and people who are not familiar with GitHub **will have a hard time** pushing their changes to the repository.
So this is a way to help begineers. :)

Suggestions on what more I could do to improve the "Contributing" section are welcome.

### CODE REVIEW

**Comment by ExclusiveKnight** [comment]
I'm a bit of a noob when it comes to GitHub. I reverted the commit above this message so it doesn't get merged into the main repo.

**Comment by ExclusiveKnight** [comment]
That is my **final commit**, I need feedback on the latest commit so this can finally be pushed to main.

**Review by lawin666 — APPROVED** [review]

**Comment by ExclusiveKnight** [comment]
Ready to merge?

**Review by xix-xeaon — CHANGES_REQUESTED** [review]
We shouldn't depend on a patch version, make the requirement 4.2 and I think this can be merged.

**Comment by ExclusiveKnight** [comment]
Will do.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
 # IT23-game
 Vampire Survivor clone by students of NTI Gymnasiet in Malmö
+
+## Requirements
+- [Godot Engine](https://godotengine.org/download/windows/)
```
[the remark this answers is not in the document]
Comment (ExclusiveKnight) [reply README.md]: Will fix that in my commit, don't worry about it lol

**Comment by ExclusiveKnight** [comment]
Fixed.

**Review by xix-xeaon — APPROVED** [review]

**Pull request merged**